openbb_sdk/providers/alpha_vantage/openbb_alpha_vantage/models/stock_eod.py

```diff
@@ -0,0 +1,217 @@
+"""Alpha Vantage Stock End of Day fetcher."""
+
+
+from datetime import datetime
+from typing import Any, Dict, List, Literal, Optional, get_args
+
+import pandas as pd
+from dateutil.relativedelta import relativedelta
+from openbb_provider.abstract.fetcher import Fetcher
+from openbb_provider.standard_models.stock_eod import StockEODData, StockEODQueryParams
+from openbb_provider.utils.descriptions import DATA_DESCRIPTIONS
+from openbb_provider.utils.helpers import get_querystring
+from pydantic import Field, NonNegativeFloat, PositiveFloat, root_validator, validator
+
+
+class AlphaVantageStockEODQueryParams(StockEODQueryParams):
+    """Alpha Vantage Stock End of Day Query.
+
+    Source: https://www.alphavantage.co/documentation/
+    """
+
+    function: Literal[
+        "TIME_SERIES_INTRADAY",
+        "TIME_SERIES_DAILY",
+        "TIME_SERIES_WEEKLY",
+        "TIME_SERIES_MONTHLY",
+    ] = Field(
+        description="The time series of your choice. ",
+        default="TIME_SERIES_DAILY",
+    )
+    interval: Optional[Literal["1min", "5min", "15min", "30min", "60min"]] = Field(
+        description="The interval between two consecutive data points in the time series.",
+        default="60min",
+        available_on_functions=["TIME_SERIES_INTRADAY"],
+        required_on_functions=["TIME_SERIES_INTRADAY"],
+    )
+    adjusted: Optional[bool] = Field(
+        description="Output time series is adjusted by historical split and dividend events.",
+        default=True,
+        available_on_functions=["TIME_SERIES_INTRADAY"],
```

Comment:
Adjusted is funny, this param is actually only valid for non-intraday endpoints and it doesn't really adjust the time series.  Instead, it adds another column "Adj Close" and all other columns remain unadjusted.  In the current Terminal, this breaks the candle charts because one column is not aligned with the others.

In a dream world, `adjusted=True` would only split-adjust prices and volume, leaving dividend adjustments out of the equation entirely.  Dividend adjustments aren't a reflection of the real price, just an assumption for valuation models.

Reply:
This can be worked on in the `transform_data` function. Not an issue atm.